src/olympia/hero/models.py

```diff
@@ -174,14 +174,25 @@ class PrimaryHero(ModelBase):
         'HTML or special tags. Will be translated.',
     )
     enabled = models.BooleanField(db_index=True, default=False)
-    # TODO: promotedaddon; primaryhero refactor (Write PR)
```

Comment:
To make this easier to land, let's **not** remove the promoted_addon field in this push, but in the next one. All of the migrations since the last push to production will run sequentially once the code is landed. The migration in it's current state is not reversible since you are dropping the field in the end. There is no need to remove that field now, we can just leave it and you can add it to the final clean up PR.